examples/vision/oml_omniglot.py

```diff
@@ -89,31 +99,41 @@ def main(
         meta_train_accuracy = 0.0
         meta_valid_error = 0.0
         meta_valid_accuracy = 0.0
+
+        learner = maml.clone()
```

Comment:
Moved clone outside the task loop.

Reply:
To not clone the entire network and make the representation/adaptation difference explicit, we can do something like:
in the inner loop:
```
representation_net = self.model.representation
adaptation_net = self.model.adaptation
# inside the inner loop
output = adaptation_net(representation_net(x))
...
inner_step(adaptation_net) # only updates the adaptation net, without calling backward 
```
in the outer loop:
```
output = adaptation_net(representation_net(x))
...
loss.backward()
opt.step() # updates the entire network, backpropagating all the way to the original network
```